test/sing_for_needs/causes/causes_test.exs

```diff
@@ -0,0 +1,108 @@
+defmodule SingForNeeds.CausesTest do
+  @moduledoc """
+    Unit tests for Causes context
+  """
+  use SingForNeeds.DataCase
+
+  alias SingForNeeds.Artists
+  alias SingForNeeds.Causes
+
+  describe "causes" do
+    alias SingForNeeds.Causes.Cause
+
+    @valid_attrs %{
+      description: "Go Blue description",
+      end_date: "2010-10-17",
+      start_date: "2010-09-17",
+      target_amount: 30_000,
+      raised_amount: 3000,
+      sponsor: "unicef",
+      name: "Go Blue on World Children's Day"
+    }
+    @update_attrs %{
+      description: "some updated description",
+      end_date: "2011-05-18",
+      name: "some updated name"
+    }
+    @invalid_attrs %{description: nil, end_date: nil, name: nil}
+
+    def cause_fixture(attrs \\ %{}) do
```

Comment:
I don't like these. You've got a method `cause_fixture/1` that performs a mutating operation. The name isn't clearly indicating what it's doing